### PR TITLE
Project sample cleaned

### PR DESCRIPTION
**Pull request by vquach2404**
Got rid of irrelevant files and changed copyright headers.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Geometry/Project/StackView.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import UIKit
+import ArcGIS
+
+class StackView: UIStackView {
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/StackView.swift]: Can this view be renamed to something related to this sample?

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#Project
+Project a point to another spatial reference.
+
+Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the AGSGeometryEngine.Intersection function. This function takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use AGSGeometryEngine.project to convert the geometries so they match.
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Can you format the API - class names, methods etc?

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+![Project App](project.png)
+
+##How to use the sample
+Click anywhere on the map. A callout will display the clicked location's coordinate in the original (basemap's) spatial reference and in the projected spatial reference.
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Click or tap?

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+##How it works
+To project a geometry to another spatial reference: Use the AGSGeometryEngine.projectGeometry function, passing in the chosen coordinates to find the spatial reference coordinates.
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: - format method
- should be a "method" instead of a "function"

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+##How to use the sample
+Click anywhere on the map. A callout will display the clicked location's coordinate in the original (basemap's) spatial reference and in the projected spatial reference.
+
+##How it works
+To project a geometry to another spatial reference: Use the AGSGeometryEngine.projectGeometry function, passing in the chosen coordinates to find the spatial reference coordinates. 
+
+##Relavant API
+* AGSGeometryEngine
+* AGSPoint
+* AGSSpatialReference
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Format API

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+###Tags
+Edit and Manage Data
+AGSGeometryEngine
+AGSSpatialReference
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: I think tags are usually comma separated

File: arcgis-ios-sdk-samples/Geometry/Project/README.md (hunk 5 above)
Comment (mikewilburn) [reply arcgis-ios-sdk-samples/Geometry/Project/README.md]: @sbaskaran we've actually been making a conscious decision to not format the APIs here with backticks as part of an effort to standardize our READMEs. I'll chat with @vquach2404 to clear this up and some other aspects related to this file.

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+![Project App](project.png)
+
+##How to use the sample
+Click anywhere on the map. A callout will display the clicked location's coordinate in the original (basemap's) spatial reference and in the projected spatial reference.
+
+##How it works
+To project a geometry to another spatial reference: Use the AGSGeometryEngine.projectGeometry function, passing in the chosen coordinates to find the spatial reference coordinates. 
+
+##Relavant API
```
Comment (mikewilburn) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Typo: Relevant API

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+#Project
+Project a point to another spatial reference.
```
Comment (mikewilburn) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: A couple of formatting considerations applicable to all headings here -- 
1) try adding a space between `#` and `Project` to ensure this becomes a heading
2) add a new line between the heading and the text below to ensure these don't get conflated to a single line

**Comment by sbaskaran** [comment]
@philium can you also please review?

File: arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+        
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["ProjectViewController", "ProjectStackView"]
+        
+        self.map = AGSMap(basemap: .nationalGeographic())
+        self.mapView.map = self.map
+        self.mapView.touchDelegate = self
+        self.mapView.setViewpointCenter(AGSPoint(x: -1.2e7, y: 5e6, spatialReference: .webMercator()), scale: 4e7)
+    }
+    
+    private func createGraphics(coord: AGSPoint) {
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift]: Wonder if we need three methods for creating and adding graphics overlay and graphics? Even so, could you maybe tweak the names a bit? For instance - creating is not the same as creating and adding graphics/overlay to mapView. 

I also think it's fine if we have to initialize the graphics overlay and add it to the mapView in `viewDidLoad()` and then you could just have one method for creating and adding graphics to the overlay. @philium might have a say in what is typically followed for samples.

File: arcgis-ios-sdk-samples/Geometry/Project/README.md (hunk 5 above)
Comment (sbaskaran) [reply arcgis-ios-sdk-samples/Geometry/Project/README.md]: @mikewilburn Ok, thanks

File: arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+        createGraphicsOverlay()
+        createPointGraphics(point: coord)
+    }
+    
+    private func createGraphicsOverlay() {
+        graphicsOverlay = AGSGraphicsOverlay()
+        self.mapView.graphicsOverlays.add(graphicsOverlay as Any)
+    }
+    private func createPointGraphics (point: AGSPoint) {
```
[the remark this answers is not in the document]
Comment (philium) [reply arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift]: I also meant to point out that there shouldn't be a space between the function name and the opening parenthesis:

```suggestion
    private func createPointGraphics(point: AGSPoint) {
```

File: arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+        let pointSymbol = AGSSimpleMarkerSymbol(style: .circle, color: .red, size: 5.0)
+        pointSymbol.outline = AGSSimpleLineSymbol(style: .solid, color: .red, width: 2.0)
+        let pointGraphic = AGSGraphic(geometry: point, symbol: pointSymbol, attributes: nil)
```
[the remark this answers is not in the document]
Comment (vquach2404) [reply arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift]: I don't really follow. I tried to omit the parameter but Xcode did not accept that.

[the remark this answers is not in the document]
Comment (philium) [reply arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift]: This works for me:

```suggestion
        let pointGraphic = AGSGraphic(geometry: coord, symbol: pointSymbol)
```

Not something you have to change, but it is a nice convenience.

**Comment by vquach2404** [comment]
Ready for a re-review. I tried to resize the screenshot of the readme as suggested but it looks different from other samples. Please let me know if I misunderstood the instructions.

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+
+## Relevant API
+
+* AGSGeometryEngine
+* AGSPoint
+* AGSSpatialReference
+
+## Additional information
+
+In cases where the the output spatial reference uses a different geographic coordinate system than that of the input spatial reference, see the GeometryEngine.project method that additionally takes in a DatumTransformation parameter.
```
Comment (mikewilburn) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Don't forget to use the fully-qualified symbol name when introducing this method here:

GeometryEngine.project --> `class AGSGeometryEngine.projectGeometry(_:to:datumTransformation:)`

Given the fact that this method name implies the datum transformation, I think you can leave out much of the second part of the sentence.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist [hunk 13]
```diff
@@ -1219,22 +1227,6 @@
 				<key>descriptionText</key>
 				<string>Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).</string>
 			</dict>
-			<dict>
```
Comment (philium) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: This change removes the Authenticate with OAuth and Token Authentication samples from the UI. Please revert it.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist [hunk 14]
```diff
@@ -1418,10 +1410,6 @@
 				<string>ChooseCameraController</string>
 				<key>descriptionText</key>
 				<string>Control the behavior of the camera in a scene.</string>
-				<key>dependency</key>
```
Comment (philium) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: This dependency is needed for the Choose Camera Controller sample. Please revert this change.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist [hunk 15]
```diff
@@ -969,6 +969,14 @@
 		<string>Geometry</string>
 		<key>children</key>
 		<array>
+			<dict>
+				<key>displayName</key>
+				<string>Project</string>
+				<key>storyboardName</key>
+				<string>Project</string>
+				<key>descriptionText</key>
+				<string>The sample shows how to project a point from decimal degrees (latitude and longitude values) to the spatial reference of the map</string>
```
Comment (philium) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: The description should match the one in the README:

```suggestion
				<string>Project a point to another spatial reference.</string>
```

File: arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+    @IBOutlet private weak var mapView: AGSMapView! {
+        didSet {
+            //initialize the map
+            mapView.map = AGSMap(basemap: .nationalGeographic())
+            mapView.touchDelegate = self
+            mapView.setViewpointCenter(AGSPoint(x: -1.2e7, y: 5e6, spatialReference: .webMercator()), scale: 4e7)
+            
+            //initialize the graphic overlay
+            mapView.graphicsOverlays.add(graphicsOverlay)
+//            self.graphicsOverlay = graphicsOverlay
```
Comment (philium) [inline arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift]: Please remove this comment.

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+
+## How to use the sample
+
+Tap anywhere on the map. A callout will display the tapped location's coordinate in the original (basemap's) spatial reference and in the projected spatial reference.
+
+## How it works
+
+1. Call the  static method, `class AGSGeometryEngine.projectGeometry(_:to:)` passing in the original `Geometry` and a `SpatialReference` to which it should be projected.
+
+
```
Comment (philium) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Please remove the extra newline here. We only put a single blank line in between sections.

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+![Project App](project.png)
+
+## How to use the sample
+
+Tap anywhere on the map. A callout will display the tapped location's coordinate in the original (basemap's) spatial reference and in the projected spatial reference.
+
+## How it works
+
+1. Call the  static method, `class AGSGeometryEngine.projectGeometry(_:to:)` passing in the original `Geometry` and a `SpatialReference` to which it should be projected.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: `Geometry` and `SpatialReference` are not symbols in the Cocoa runtime. Please either use non-symbols ("passing in the original geometry and a spatial reference") or the actual name of the symbols ("passing in the original `AGSGeometry` and an `AGSSpatialReference`")

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist (hunk 13 above)
Comment (vquach2404) [reply arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: I'm slightly confused. From my understanding, I should add `<dict>` to line 1230, is that correct? I am unsure because if added, it doesn't seem to have an end tag.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist (hunk 14 above)
Comment (vquach2404) [reply arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: I am also confused here because there's no array of strings that follows this line if I were to add it. Can this line of code work without a succeeding array of strings?

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist (hunk 13 above)
Comment (philium) [reply arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: I apologize for the confusion; I was referring to this block of changes. In case you didn't know, additions in your branch that do not exist in the target branch appear in green and removals from your branch that exist in the target branch appear in red. This group of changes reflect lines that were somehow removed from your branch. If they were to be merged with this PR, those lines would no longer exist in `v.next`.

Does that make sense?

Comment (vquach2404) [reply arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: Yes, that clarifies things, thanks!

File: arcgis-ios-sdk-samples.xcodeproj/project.pbxproj [hunk 19]
```diff
@@ -4534,7 +4558,7 @@
 					"@executable_path/Frameworks",
 				);
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.esri.arcgis-ios-sdk-samples";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.esri.arcgis-ios-sdk-samplesV";
```
Comment (saratchandrakarumuri) [inline arcgis-ios-sdk-samples.xcodeproj/project.pbxproj]: Is there a reason why the bundle identifier is changes?

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+
+## Use case
+
+Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `class AGSGeometryEngine.intersection(ofGeometry1:geometry2:)` method. This method takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `class AGSGeometryEngine.projectGeometry(_:to:)` to convert the geometries so they match.
+
+![Project App](project.png)
+
+## How to use the sample
+
+Tap anywhere on the map. A callout will display the tapped location's coordinate in the original (basemap's) spatial reference and in the projected spatial reference.
```
Comment (saratchandrakarumuri) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: I think it is better to mention the output spatial reference (wgs84) somewhere in the readme doc

File: arcgis-ios-sdk-samples.xcodeproj/project.pbxproj (hunk 19 above)
Comment (vquach2404) [reply arcgis-ios-sdk-samples.xcodeproj/project.pbxproj]: It was Divesh's solution that allowed me to deploy this app on a device. I'm currently resolving this situation with Nick. But I had reverted the bundle identifier in the general tab in my most recent commit, do I have to change the bundle identifier somewhere else in order to fix this?

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+
+## Use case
+
+Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `class AGSGeometryEngine.intersection(ofGeometry1:geometry2:)` method. This method takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `class AGSGeometryEngine.projectGeometry(_:to:)` to convert the geometries so they match.
+
+![Project App](project.png)
+
+## How to use the sample
+
+Tap anywhere on the map. A callout will display the tapped location's coordinate in the original (basemap's) spatial reference (wgs84) and in the projected spatial reference.
```
Comment (saratchandrakarumuri) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]: Original spatial reference is `webMercator` and the projected spatial reference is `wgs84`

Comment (vquach2404) [reply arcgis-ios-sdk-samples/Geometry/Project/README.md]: For clarity, should I replace that portion of the sentence with "spatial reference `webMercator` and in the projected spatial reference `wgs84` ?"

Comment (saratchandrakarumuri) [reply arcgis-ios-sdk-samples/Geometry/Project/README.md]: Sounds good to me

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist (hunk 13 above)
Comment (philium) [reply arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: Thank you for adding the removed lines back in. However, the indentation of those lines has now changed (which is why GitHub is still showing them as modified). Please fix the indentation again. When you're done and you've pushed your changes, please look at the PR and make sure that the only changes to this file are the eight new lines you added for the new sample.

File: arcgis-ios-sdk-samples/Geometry/Project/README.md [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+
+## Relevant API
+
+* AGSGeometryEngine
+* AGSPoint
+* AGSSpatialReference
+
+## Additional information
+
+In cases where the the output spatial reference uses a different geographic coordinate system than that of the input spatial reference, see the `class AGSGeometryEngine.projectGeometry(_:to:datumTransformation:)`  method.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Geometry/Project/README.md]:
```suggestion
In cases where the the output spatial reference uses a different geographic coordinate system than that of the input spatial reference, see the `class AGSGeometryEngine.projectGeometry(_:to:datumTransformation:)` method.
```

**Comment by vquach2404** [comment]
> Thank you for updating the screenshot. The content looks good, but it is kind of small (low resolution). What device/simulator did you take the screenshot from? iPhone X, Xs, Xs Max, or Xr?

I took the screenshot with an iPhone Xr.

**Comment by philium** [comment]
> I took the screenshot with an iPhone Xr.

The pixel resolution of an iPhone Xr is 1,792x828. The width is evenly divisible by 3, yielding a pixel resolution of 597x276 (preserving aspect ratio), 33% of the original. That's what I showed in the Adjust Size… screenshot of Preview. The pixel resolution of the screenshot you updated is 217x111, about 13% of the original. I think that's a little too small.

**Comment by vquach2404** [comment]
Thanks for all the help everyone!

**Comment by vquach2404** [comment]
> LGTM! ✅

Let's Get This Merged?

**Comment by mikewilburn** [comment]
@vquach2404 we usually use it to mean 'Looks Good To Me' 😆 
But I suppose now it's just an acronym that kills two birds with one stone. :)
